client/shell/mdb_code_versioning.py

```diff
@@ -17,10 +17,10 @@ def version(version_config):
     session = requests.Session()
     session.headers['Authorization'] = 'token %s' \
         % version_config[constants.ACCESS_TOKEN_KEY]
-    
+
     # check if the repo exists, if not create it
     repo_exists_url = 'https://api.github.com/repos/%s/%s' % \
-        (version_config[constants.GIT_USERNAME_KEY], 
+        (version_config[constants.GIT_USERNAME_KEY],
```

Comment:
If the constants are removed -- I'm unclear on what we are doing there. Is constants going to be around?